app/routes/report/kbMatches.js

```diff
@@ -76,12 +76,33 @@ router.route('/')
     }
 
     try {
+      const therapeuticAssociationFilter = {
+        [Op.or]: [{iprEvidenceLevel: ['IPR-A', 'IPR-B']}],
+        category: 'therapeutic',
+      };
```

Comment:
This is only for the rapid report, right?

Reply:
Yes, only activated if you use the rapidTable query in kb-matches endpoint